custom.go

```diff
@@ -36,7 +36,7 @@ type CustomConfig struct {
 }
 
 // Called on system startup.
-func OnStart(ctx *Context, params Parameters) {
+func OnStart(ctx *Context, params Parameters) *sql.DB{
```

Comment:
I don't think this commit is needed. The DB pointer is already returned as part of the Context object...

Reply:
updated. Use the db pointer from ctx